### PR TITLE
Create brand_impersonation_siriusxm.yml

### PR DESCRIPTION
**Pull request by aidenmitchell**
# Description

Impersonation of the broadcasting corporation SiriusXM.

# Associated samples

- https://platform.sublime.security/messages/a68684e74cddcaeb5e66ca2ac973f079ddf57a5bfeaa3276551f8581518bb40e

### CODE REVIEW

**Comment by aidenmitchell** [comment]
Looks good in test-rules, low volume

File: detection-rules/brand_impersonation_siriusxm.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: "Brand impersonation: SiriusXM"
```
Comment (zoomequipd) [inline detection-rules/brand_impersonation_siriusxm.yml:1]:
```suggestion
name: "Brand Impersonation: SiriusXM"
```

File: detection-rules/brand_impersonation_siriusxm.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  type.inbound
+  and (
+    strings.ilike(sender.display_name, '*siriusxm*')
+    or strings.ilevenshtein(sender.display_name, 'siriusxm') <= 1
+    or strings.ilike(sender.email.domain.domain, '*siriusxm*')
+  )
+  and (
+    sender.email.domain.root_domain not in ('siriusxm.com')
+    or (
+      sender.email.domain.root_domain in ('siriusxm.com')
```
Comment (zoomequipd) [inline detection-rules/brand_impersonation_siriusxm.yml:15]:
```suggestion
      sender.email.domain.root_domain in ('siriusxm.com', 'siriusxmmedia.com', 'siriusxm.ca')
```

File: detection-rules/brand_impersonation_siriusxm.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+severity: "medium"
+source: |
+  type.inbound
+  and (
+    strings.ilike(sender.display_name, '*siriusxm*')
+    or strings.ilevenshtein(sender.display_name, 'siriusxm') <= 1
+    or strings.ilike(sender.email.domain.domain, '*siriusxm*')
+  )
+  and (
+    sender.email.domain.root_domain not in ('siriusxm.com')
```
Comment (zoomequipd) [inline detection-rules/brand_impersonation_siriusxm.yml:13]:
```suggestion
    sender.email.domain.root_domain not in ('siriusxm.com', 'siriusxmmedia.com', 'siriusxm.ca')
```

**Review by zoomequipd — CHANGES_REQUESTED** [review]
some feedback based on hunts

**Review by zoomequipd — APPROVED** [review]

**Pull request merged**